README.md

```diff
@@ -87,7 +87,7 @@ When updating this, also update:
 - .github/workflows/lint.yml
 -->
 
-The Minimum Supported Rust Version (MSRV) of this project is [1.79.0](https://blog.rust-lang.org/2024/06/13/Rust-1.79.0.html).
+The Minimum Supported Rust Version (MSRV) of this project is [1.80.0](https://blog.rust-lang.org/2024/06/13/Rust-1.80.0.html).
```

Comment:
you need to update the blogpost link too. 
@mattsse should we remove this link?

Reply:
now that's updated, maybe next time :D